### DISCUSSION TITLE
[11 gennaio 2022 - A] Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#198] [Esami]
**Esercizio 6 (2+2 punti)**

Dati i numeri in rappresentazione esadecimale IEEE 754 half-precision $X = 4080$ e $Y = BD00$,

(a) eseguire l’operazione $X+Y$ secondo lo standard IEEE 754
(b) rappresentare il risultato in notazione esadecimale, poi convertire operandi e risultato in base dieci.

**Comment by Elia-Belli** [comment]
- $X = 4080_{16} = 0\ 10000\ 0010000000 = +2^1\ 1.001 $
  $Y = BD00_{16} = 1\ 01111\ 0100000000 = -2^0\ 1.01 = -2^1\ 0.101$
  $- Y = 1.010 +1 = 1.011$ (siccome negativo per la somma converto in Ca2)
 $01.001 +$
 $11.011 =$
 $00.100$
 $Z = 2^1\ 0.1 = 2^0\ 1 = 0\ 01111\ 0000000000 = 3C00_{16}$

- $X = +2.25_{10}$
$Y = -1.25_{10}$
$Z = 1_{10}$

**Comment by FeddyLix17** [comment]
corretto
